### PR TITLE
Style updates, remove classNames dependency, Timeline

### PR DESCRIPTION
**Pull request by Paterma**
Combined a few tickets here- style updates to match PDF, remove classNames dep, notification banners, and Timeline. 
I noticed while making the horizontal scroll bar on the timeline sticky, that the Contacts toolbar scrolls with the timeline, and the vertical scroll bar goes to the top of that bar. It needs to be moved out of that main container, which will require a lot of styling updates. I'm creating another ticket for this. 
The other thing I noticed is there are some css styles that don't actually do anything. I think a whole comb-over of all the css files would be very helpful, and taking some complexity out of the classnames/removing as many as possible (similar to what we did on TT&C monitor). I'm making another ticket for this as well.

### CODE REVIEW

File: src/components/GlobalStatusBar/GlobalStatusBar.tsx [hunk 1]
```diff
@@ -26,35 +28,53 @@ const GlobalStatusBar = () => {
...
+            <RuxMenuItem>GRM Schedule</RuxMenuItem>
+            <RuxMenuItemDivider />
+            <RuxMenuItem>Preferences...</RuxMenuItem>
+            <RuxMenuItem>Sign Out...</RuxMenuItem>
+          </RuxMenu>
+        </RuxPopUp>
+
+        <RuxClock />
+
+        <RuxMonitoringProgressIcon
```
Comment (Paterma) [inline src/components/GlobalStatusBar/GlobalStatusBar.tsx:70]: This is not right aligned in the PDF, but, I think it looks better how Brian had it. Thoughts? We can let design decide

**Review by Full-lifey — CHANGES_REQUESTED** [review]
Style issues shown in the screenshots.

I will review code after these are resolved.

**Comment by Full-lifey** [comment]
Segmented button is in a static location also and not responsive to the right side.
image

File: src/common/AstroReactTable/AstroReactTable.tsx [hunk 2]
```diff
@@ -33,17 +32,20 @@ export const AstroReactTable = ({
...
-              'Astro-react-table__sorted': isSortable && !!column.getIsSorted(),
-              'Astro-react-table__sortable': isSortable,
-            })}
+            className={
+              isSortable && !!column.getIsSorted()
+                ? 'Astro-react-table__col Astro-react-table__sorted'
+                : 'Astro-react-table__col' || isSortable
+                ? 'Astro-react-table__col Astro-react-table__sortable'
+                : 'Astro-react-table__col'
+            }
```
Comment (Full-lifey) [inline src/common/AstroReactTable/AstroReactTable.tsx:41]: You can use a template literal so you don't have to repeat the `Astro-react-table__col` class in your ternaries.

```suggestion
            className={`Astro-react-table__col ${
              isSortable && !!column.getIsSorted()
                ? 'Astro-react-table__sorted'
                : isSortable
                ? 'Astro-react-table__sortable'
                : ''
            }`}

```

File: src/common/AstroReactTable/AstroReactTable.tsx [hunk 3]
```diff
@@ -53,26 +55,31 @@ export const AstroReactTable = ({
...
+            <div
+              key={id}
+              onClick={() => handleRowClick(original)}
+              className={
+                handleIsSelected(original)
+                  ? 'Astro-react-table__row Astro-react-table__selected'
+                  : 'Astro-react-table__row' || !!onRowClick
+                  ? 'Astro-react-table__row Astro-react-table__selectable'
+                  : 'Astro-react-table__row'
+              }
```
Comment (Full-lifey) [inline src/common/AstroReactTable/AstroReactTable.tsx:70]: Same here, a template literal will simplify this a bit:

```suggestion
              className={`Astro-react-table__row ${
                handleIsSelected(original)
                  ? 'Astro-react-table__selected'
                  : !!onRowClick
                  ? 'Astro-react-table__selectable'
                  : ''
              }`}
```

File: src/components/App.tsx [hunk 4]
```diff
@@ -64,16 +63,16 @@ const App = () => {
...
         <SearchBar />
         <RuxContainer className='App-main__container'>
           <ContactsHeader {...{ isOpen, handleAction }} />
 
-          <div className={classNames('App-main__left-panel', { isOpen })}>
+          <div
+            className={
+              isOpen ? 'App-main__left-panel isOpen' : 'App-main__left-panel'
+            }
+          >
```
Comment (Full-lifey) [inline src/components/App.tsx:75]:
```suggestion
          <div className={`App-main__left-panel ${isOpen ? 'isOpen' : ''}`}>
```

File: src/components/App.tsx [hunk 5]
```diff
@@ -83,8 +82,12 @@ const App = () => {
             )}
           </div>
 
-          <aside className={classNames('App-main__right-panel', { isOpen })}>
-            {rigthPanel}
+          <aside
+            className={
+              isOpen ? 'App-main__right-panel isOpen' : 'App-main__right-panel'
+            }
+          >
```
Comment (Full-lifey) [inline src/components/App.tsx:89]:
```suggestion
          <aside className={`App-main__right-panel ${isOpen ? 'isOpen' : ''}`}>
```

File: src/components/GlobalStatusBar/GlobalStatusBar.tsx (hunk 1 above)
Comment (Full-lifey) [reply src/components/GlobalStatusBar/GlobalStatusBar.tsx:70]: It's good

**Review by Full-lifey — CHANGES_REQUESTED** [review]
Great work! I just noticed that some JS template literals will clean up some repeating class names.

**Pull request merged**